### DISCUSSION TITLE
Feature request: Write pages honouring honourable people

### DISCUSSION THREAD

**Discussion by SheepTester** [Gamepro5/Gamepro5.com#3] [Implemented Ideas]
Such as,

- Master ovinus
- Master TheVoidSurvivor
- Master almundmilk
- Master based
- Kevin
- Songwriter Nichodon
- Charismatic Henry

**Comment by Gamepro5** [comment]
I am not familiar with "Master based" and "Kevin". But the other ones will do!

**Reply by SheepTester** [reply]
NTBased angers
